focus-android/docs/Home.md

```diff
@@ -15,12 +15,12 @@ https://www.mozilla.org/firefox/focus/
 
 * Google Play - Firefox Focus: https://play.google.com/store/apps/details?id=org.mozilla.focus
 * Google Play - Firefox Klar: https://play.google.com/store/apps/details?id=org.mozilla.klar
-* APKs: https://github.com/mozilla-mobile/focus-android/releases
+* APKs: https://github.com/mozilla-mobile/firefox-android/releases
 
 ## Contribute
 
-* [Contributing (Writing code, translating the app, testing the app)](https://github.com/mozilla-mobile/shared-docs/blob/master/android/CONTRIBUTING.md)
-* [List of good first issues for new contributors](https://github.com/mozilla-mobile/focus-android/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+* [Contributing (Writing code, translating the app, testing the app)](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/CONTRIBUTING.md)
+* [List of good first issues for new contributors](https://bugzilla.mozilla.org/describecomponents.cgi?product=Focus)
```

Comment:
I am happy with where the link points, but could we update the link text to be "List of issues"

Reply:
Sure